packages/fluent/scss/toolbar/_theme.scss

```diff
@@ -22,6 +19,35 @@
 
     }
 
+    // Solid Toolbar
+    .k-toolbar-solid {
+        @include fill (
+            var( --kendo-toolbar-text, #{$kendo-toolbar-text} ),
+            var( --kendo-toolbar-bg, #{$kendo-toolbar-bg} ),
+            var( --kendo-toolbar-border, #{$kendo-toolbar-border} )
+        );
+    }
+
+    // Outline Toolbar
+    .k-toolbar-outline {
+        @include fill (
+            $color: var( --kendo-toolbar-outline-text, #{$kendo-toolbar-outline-text} ),
+            $border: var( --kendo-toolbar-outline-border, #{$kendo-toolbar-outline-border} )
+        );
+    }
+
+    // Flat Toolbar
+    .k-toolbar-flat {
+        @include fill (
+            $color: var( --kendo-toolbar-flat-text, #{$kendo-toolbar-flat-text} ),
+            $border: var( --kendo-toolbar-flat-border, #{$kendo-toolbar-flat-border} )
+        );
+
+        > .k-separator {
+            border-inline-color: $kendo-toolbar-flat-border;
```

Comment:
The correct syntax here should be: border-inline-color: var( --kendo-toolbar-flat-border, #{$kendo-toolbar-flat-border} );